src/mmw/requirements/base.txt

```diff
@@ -18,3 +18,4 @@ rollbar==0.13.8
 retry==0.9.1
 python-dateutil==2.6.0
 suds==0.4
+django-celery-results==1.0.1
```

Comment:
I don't think we need this. Celery task results are stored manually via [`core/job`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/core/models.py), and we don't interact with them via ORM at all.

Reply:
I'll try out dropping this again.

Reply:
Ok, I tried out dropping this and the destroying & rebuilding the worker and the upgrade won't work from that state without some additional change because it looks like the worker's version of Celery sticks at 3.1.25.

I think that at the very least `django-celery-results` might be forcing an upgrade to the worker's Celery installation that's not currently captured in the Ansible rules. Can't speak to whether we need the library altogether, but we do need to ensure that there's an explicit rule to upgrade the worker in addition to the app.

Reply:
Sounds like the `model-my-watershed.celery` is not applying cleanly to `worker` or that something is downgrading it after the dependency is installed.  Definitely want to make sure we're getting the version installed from `model-my-watershed.celery` and not transitively from another library.